sdk/python/feast/registry_server.py

```diff
@@ -36,14 +36,15 @@ def __init__(self, registry: BaseRegistry) -> None:
         self.proxied_registry = registry
 
     def ApplyEntity(self, request: RegistryServer_pb2.ApplyEntityRequest, context):
-        assert_permissions(
-            resource=self.proxied_registry.apply_entity(
-                entity=Entity.from_proto(request.entity),
-                project=request.project,
-                commit=request.commit,
+        self.proxied_registry.apply_entity(
```

Comment:
linter fails, a cast is needed in all such changes (sorry for not noticing before:
```py
            entity=cast(
                Entity,
                assert_permissions(
                    resource=Entity.from_proto(request.entity),
                    actions=CRUD,
                ),
            ),
```

Reply:
Fixed